### ISSUE TITLE
Windows Binary Operation of Sloth

### ISSUE THREAD

**Issue by ahaw021** [slok/sloth#135]
hi @slok 

I really like the sloth. Especially the use of OpenSLO and other providers. 

I am having a few challenges making it work on windows and I am hoping it's just missing documentation. 

A) I assume this error is with SLI Plugins rather than the actual spec. It's difficult to understand the error and I had to lint the sample YAML a few times

![image](https://user-images.githubusercontent.com/26753809/129002195-1749a6fb-ae1a-41dd-be97-0965dd4769e1.png)

B) I Then followed the instructions in the SLI Plugins repo https://github.com/slok/sloth-common-sli-plugins 

![image](https://user-images.githubusercontent.com/26753809/129002460-c2c37b1d-5c90-4a5a-853a-9742f8b19d0a.png)

C) I suspect that I need a go runtime on the machine to be able to use the plugins. Is this assumption correct?

D) Is there anyway of bundling the SLI Plugins in to the binary ? 

Once again really great tool and I love the syntax

**Comment by slok** [comment]
Hi @ahaw021!

Thanks for the kind words 😄 

I didn't try myself on Windows, maybe you are the first user! 🎉 

My hypothesis is that the problem is with the file loading process and paths and not the plugins itself (not saying that this will work in windows either, however it should). It seems that is loading files as empty files (it doesn't error with a file not found error), strange 🤔.

I would need to make some tests with a windows machine first to give you an answer.

**Comment by slok** [comment]
Hey @ahaw021!

I had the chance to use a windows machine and test Sloth. I downloaded the binary and the examples and I didn't have any problems, everything works as expected:

I tested:
- Generating from spec files in the same dir.
- Generating from spec files in the other dir.
- Generating from generic Sloth specs.
- Generating from Kubernetes Sloth specs.
- Generating loading and using Pplugins.

Check these:

Generic spec and k8s spec:
![generic spec and k8s spec](https://user-images.githubusercontent.com/128204/130364268-a6364af0-8afe-4f64-a2e0-eefa46630d2d.png)

Using plugins:
![Using plugins](https://user-images.githubusercontent.com/128204/130364276-9ae2a706-7e03-49ae-923a-e3049bc50ef1.png)

All worked correctly. So my guess is that the spec files you downloaded are corrupt, empty, or something related to the file content not being correct.

I'm closing the issue, if you find any other issues reopen this or create a new one!
